core/src/main/scala/org/sciscala/simpledf/DataFrame.scala

```diff
@@ -45,176 +33,6 @@ final case class InsertError(message: String) extends Error
   def to[A](df: DFImpl)(implicit E: Encoder[DFImpl, A]): Seq[A] =
     E.encode(df)
   def items(df: DFImpl): Array[(String, Column[_])]
+  def iterrows(df: DFImpl)(implicit encoder: Encoder[DFImpl, Row]): Seq[(String, Row)]
 }
 
-object DataFrame {
```

Comment:
This is Dataframe's companion object, removing code from here may have negative impact both discoverability and type inference. Its also recommended for type-classes to define their instances inside a companion object.

```Companion object: popular place to put your implicit instances val/defs. Compiler will look at Companion object members by default, so that we do not need to explicitly import implicit values in scope.```

https://medium.com/se-notes-by-alexey-novakov/of-scala-type-classes-6647c48e39d9#more_on_implicits

Reply:
Added a commit with changes to implicits.
At same time , I fixed arrowspec tests using simulacrum ops